pyspecProcScripts/phasing.py

```diff
@@ -221,7 +264,7 @@ def filter_ranges(B, A):
         return [np.array(b) for b in B if
                 any(b[0] <= a[0] and b[1] >= a[1] for a in A)]
     peakrange = filter_ranges(wide_ranges, narrow_ranges)
-    assert len(peakrange) == 1
+    assert (len(peakrange) == 1),"""Your wide range is too small try decreasing the peak_lower_thresh"""
```

Comment:
That's not what this means.  Rather, have it say it found two peaks, and give the location of both peaks.

Reply:
If this is not working, I think I'll need to help by seeing a specific example of a dataset where it doesn't work, with MWE showing that it doesn't work

Reply:
pushed a notebook FIDfromEcho_fail.py